app/controllers/to_do_items_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ToDoItemsController < ApplicationController
+
+  def new
+    list = List.find params[:id]
+    @item = list.items.new
+
+    render partial: 'to_do_items/form', locals: {item: @item}
+  end
```

Comment:
@vlasiak, just a heads up: as discussed, `form` partial could be renamed into `new` and be used as the view of the action.